### ISSUE TITLE
What is the difference between "Next-js-Boilerplate" vs "SaaS-Boilerplate? When to use what?

### ISSUE THREAD

**Issue by avibathula** [ixartz/SaaS-Boilerplate#37]

**Comment by ixartz** [comment]
Yes, SaaS Boilerplate is a superset of Next.js Boilerplate. Just with more features for someone who wants to build a SaaS, features like Multi-tenancy, Landing page for SaaS, User dashboard for SaaS.

Otherwise, it's exactly the same.

SaaS Boilerplate also uses PGlite.

Next.js Boilerplate can be used when someone want to build other application (not SaaS).

**Comment by avibathula** [comment]
Kudos @ixartz - I have been researching on a NextJS boilerplate for building a SaaS idea and I love what you have done here compared to many others. I am going to get started to experiment this today.

**Comment by ixartz** [comment]
Thank you for your message!

If you have any suggestion and feedback during your experiment, I'm totally open to it.
